DESIGN-README.md

```diff
@@ -0,0 +1,37 @@
+# Getting Started
+
+## Design Tools
+Design tools we use:
+* Sketch
+* Invision
+
+## Design Files
+* Sketch file (request link)
+* [Icon Set](https://www.emojione.com/developers/download?key=8YNLFBYZAB47ST8ZDZV3)
+
+## Styleguide/Branding
+This application was designed using an Atomic design system.
+
+* [Styleguide](http://demo.getcalfresh.org/styleguide)
+* [Styleguide code](https://github.com/codeforamerica/gcf-backend/tree/master/app/views/styleguides)
+
+# Day-to-Day
+
+## Communication
+Design communication is done primarily via:
+* Slack
+* Invision
+* Trello
+
+## Design Files
+Latest design files should be pushed to your branch daily to be kept up to date and accessible to other designers.
+
+## Front-End Design Contribution
+Please read the [Contributing Guidelines](https://github.com/codeforamerica/michigan-benefits/blob/master/CONTRIBUTING.md) for details on the process for code contribution and submitting pull request.
+
+## Workflow
+* A design feature or change is initiated and discussed via a Trello card.
+* All new Trello cards are added to the "Design/UI Backlog" Trello list.
+* During a weekly planning meeting, prioritized Trello cards are added to the "Next Up" Trello list. Cards in this list should have enough detail to be actionable. Top priority items are at the top of the list.
+* When someone starts working on a task, they move the Trello card into "In Progress" and assign themselves so that others know they are working on it.
+* When finished, the designer should request a review by the product manager or another designer.
```

Comment:
what is the preferred way to request a review?

Reply:
A review on Trello makes sense. I'll add that in